ignite/contrib/handlers/param_scheduler.py

```diff
@@ -402,6 +402,31 @@ def get_param(self):
         return self.start_value + ((self.end_value - self.start_value) / 2) * (1 - math.cos(math.pi * cycle_progress))
 
 
+class _ParamGroupsOptimizer:
+    """Internal tools to concat a list of parameters from different optimizers.
+    """
+
+    def __init__(self, schedulers):
+        self.schedulers = schedulers
+        self.param_groups = [s.optimizer_param_groups[0] for s in schedulers]
```

Comment:
Why only zero group `s.optimizer_param_groups[0]` ?

Reply:
Good catch. `optimizer_param_groups` is a list of parameters from legacy optimizer or `_ParamGroupsOptimizer`.

I think I have to think about all comments below to push a better solution...